dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -397,7 +399,13 @@ MULTI_USER_REPLACEMENT_STRING="          - name: \"CHE_WORKSPACE_LOGS\"
           - name: \"CHE_KEYCLOAK_CLIENT__ID\"
             value: \"${CHE_KEYCLOAK_CLIENT__ID}\"
           - name: \"CHE_HOST\"
-            value: \"${CHE_HOST}\""
+            value: \"${CHE_HOST}\"
+          - name: \"CHE_MULTIUSER\"
+            value: \"${CHE_MULTIUSER}\"
+          - name: \"CHE_OAUTH_GITHUB_CLIENTID\"
+            value: \"${CHE_OAUTH_GITHUB_CLIENTID}\"
```

Comment:
OAUTH credentials don't look related to multi-user case. Have you considered using it in another context?

Reply:
😱 😱 😱 +100

Reply:
I've renamed VAR from `MULTI_USER_REPLACEMENT_STRING` to `CHE_SERVER_CONFIGURATION`
because now it is not in multi / single user context and used for each usecase.

Reply:
ok